pkg/diff/util_test.go

```diff
@@ -23,7 +23,7 @@ var _ = Suite(&testUtilSuite{})
 
 type testUtilSuite struct{}
 
-func (s *testUtilSuite) TestRemoveColumns(c *C) {
+func (s *testUtilSuite) TestIgnoreColumns(c *C) {
```

Comment:
do we need to rename `removeColumns` to `ignoreColumns` in _util.go_?

Reply:
name all changed